test/algorithms/simulation/sidb/operational_domain.cpp

```diff
@@ -485,33 +483,33 @@ TEST_CASE("BDL wire operational domain computation", "[operational-domain]")
                                                                  op_domain_params, &op_domain_stats);
 
             // check if the operational domain has the correct size
-            CHECK(op_domain.operational_values.size() >= 80);
+            CHECK(op_domain.operational_values.size() <= 256);
 
             // for the selected range, all samples should be within the parameters
             check_op_domain_params_and_operational_status(op_domain, op_domain_params, std::nullopt);
 
             CHECK(mockturtle::to_seconds(op_domain_stats.time_total) > 0.0);
             CHECK(op_domain_stats.num_simulator_invocations <= 512);
-            CHECK(op_domain_stats.num_evaluated_parameter_combinations <= 512);
-            CHECK(op_domain_stats.num_operational_parameter_combinations == 80);
-            CHECK(op_domain_stats.num_non_operational_parameter_combinations <= 100);
+            CHECK(op_domain_stats.num_evaluated_parameter_combinations <= 256);
+            CHECK(op_domain_stats.num_operational_parameter_combinations <= 80);
+            CHECK(op_domain_stats.num_non_operational_parameter_combinations <= 176);
         }
         SECTION("contour_tracing")
         {
             const auto op_domain = operational_domain_contour_tracing(lat, std::vector<tt>{create_id_tt()}, 50,
                                                                       op_domain_params, &op_domain_stats);
 
             // check if the operational domain has the correct size (max 10 steps in each dimension)
-            CHECK(op_domain.operational_values.size() <= 100);
```

Comment:
@marcelwa I don't know why we chose ``100`` as the upper limit here? I guess now that this is fixed, all bugs and potential problems should be found. Thanks to @wlambooy for finding this one here!